docs/advanced/catkin_shell_verbs.rst

```diff
@@ -0,0 +1,36 @@
+Shell support in ``catkin`` command
+===================================
+
+When you source the following file, you can use bash/zsh shell functions which provide added utility.
+
+.. code-block:: shell
+
+    . /opt/ros/indigo/etc/bash_completion.d/catkin_shell_verbs.{bash,zsh}
```

Comment:
thanks, in this case, we should install `catkin_shell_verbs.bash` under `lib/python2.7/dist-packages/catkin_tools`. Is it correct? now I install this file under `share/catkin_tools`.